LedgerWallet.Tests/U2FTests.cs

```diff
@@ -12,39 +12,39 @@
 
 namespace LedgerWallet.Tests
 {
-	[Trait("U2F", "U2F")]
-	public class U2FTests
-	{
-		[Fact]
-		[Trait("Manual", "Manual")]
-		public void CanEnrollAndAuthenticate()
-		{
-			var appId = new AppId(Encoders.Hex.DecodeData("d2e42c173c857991d5e1b6c81f3e07cbb9d5f57431fe41997c9445c14ce61ec4"));
-			var challenge = Encoders.Hex.DecodeData("e6425678fbd7d3d8e311fbfb1db8d26c37cf9f16ac81c95848998a76ce3d3768");
-			U2FClient u2f = U2FClient.GetHIDU2F().First();
+    [Trait("U2F", "U2F")]
+    public class U2FTests
+    {
+        [Fact]
+        [Trait("Manual", "Manual")]
+        public async Task CanEnrollAndAuthenticate()
+        {
+            var appId = new AppId(Encoders.Hex.DecodeData("d2e42c173c857991d5e1b6c81f3e07cbb9d5f57431fe41997c9445c14ce61ec4"));
+            var challenge = Encoders.Hex.DecodeData("e6425678fbd7d3d8e311fbfb1db8d26c37cf9f16ac81c95848998a76ce3d3768");
+            var u2f = (U2FClient)await NanoSTests.GetLedgerAsync(NanoSTests.LedgerType.U2F);
```

Comment:
keep tabs

Reply:
I will revert my formatting changes here, but the original has an extra tab. I did keep the tabs. The problem is that the original is not formatted the way Visual Studio formats tabs. It's different to the other classes.

Reply:
Oops. This was a visual studio bug. I did a format for tabs, but it kept the spaces because nothing had changed.